### ISSUE TITLE
NER实体识别的问题，能否指定单个标签只对应单个实体，或者说想要实现这种方式要如何处理比较好

### ISSUE THREAD

**Issue by WMeng1** [bojone/bert4keras#504]
提问时请尽可能提供如下信息：

### 基本信息
- 你使用的**操作系统**: 
- 你使用的**Python**版本: 
- 你使用的**Tensorflow**版本: 
- 你使用的**Keras**版本: 
- 你使用的**bert4keras**版本: 
- 你使用纯**keras**还是**tf.keras**: 
- 你加载的**预训练模型**:

### 核心代码
```python
# 请在此处贴上你的核心代码。
# 请尽量只保留关键部分，不要无脑贴全部代码。
```

### 输出信息
```shell
# 请在此处贴上你的调试输出
```

### 自我尝试
不管什么问题，请先尝试自行解决，“万般努力”之下仍然无法解决再来提问。此处请贴上你的努力过程。

在实际工作中发现，标注数据中单条文本中并没有出现一个标签下存在多个实体值的情况，但实际预测出现了一个标签下有多个实体的情况，比如 地区：[北京、上海]。
想请问怎么做，或者说能不能优雅的做到在没有重复标签的前提下计算得到NER的最优路径，比如出现 地区：[北京，上海] 的情况下，选取另一条没有重复标签的路径，需要在解码器中怎么进行改动更有效呢？

    def decode(self, nodes):
        """nodes.shape=[seq_len, num_labels]
        """
        nodes[0, self.non_starts] -= np.inf
        nodes[-1, self.non_ends] -= np.inf

        labels = np.arange(self.num_labels).reshape((1, -1))
        scores = nodes[0].reshape((-1, 1))
        paths = labels
        for l in range(1, len(nodes)):
            M = scores + self.trans + nodes[l].reshape((1, -1))
            idxs = M.argmax(0)
            scores = M.max(0).reshape((-1, 1))
            paths = np.concatenate([paths[:, idxs], labels], 0)

        return paths[:, scores[:, 0].argmax()]
